lib/logger.js

```diff
@@ -1,28 +1,11 @@
-const winston = require('winston');
+const bunyan = require('bunyan');
 
 const config = require('../config');
+const packageInfo = require('../package');
 
-const logger = winston.createLogger({
-    level: config.logger.level,
-    silent: config.logger.silent,
-    format: winston.format.json(),
-    transports: [
-        new winston.transports.Console({
-            format: winston.format.combine(
-                winston.format.colorize(),
-                winston.format.timestamp(),
-                winston.format.align(),
-                winston.format.splat(),
-                winston.format.printf(info => `${info.timestamp} [${info.level}]: ${info.message}`),
-            )
-        })
-    ]
+const logger = bunyan.createLogger({
```

Comment:
what is the difference between the two? the encoding?

Reply:
rephrase: is there really no way to use winston with json, i guess? in which case, it's too much of a hassle to use one driver for dev and one for prod unfortunately

Reply:
Main difference:
- bunyan always logs as json and you can pipe it to its cli to make it human readable
- winston love to either json or human readable format, depending on the config, there's no cli.

So in bunyan I can do the following: on dev, log as json and pipe it to cli to get the human readable format. In winston I can do the following: on dev log as human readable format, on prod log as json. But this is less flexible as I can't pipe winston's output to cli on prod (if there is a reason to do it) to get human readable format from terminal. I can do it with bunyan though, that's why I considered switching to it.

Reply:
We also use bunyan at work and had no issues with it

Reply:
thanks for the thorough answer, glad we can do what we said